.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -1029,3 +1050,23 @@ Add the following to enable read access to trusted clusters
   - [ ] Clean the Application Support dir for Connect. Start the latest stable version of the app.
         Open every possible document. Close the app. Start the current alpha. Reopen the tabs. Verify that
         the app was able to reopen the tabs without any errors.
+
+[^cloud-tenant-pre-release-deployments]: Instructions for deploying a custom release to a cloud staging tenant: https://github.com/gravitational/teleport.e/blob/master/dev-deploy.md
+
+[^cloudshell-prerelease-cdn]: The AWS integration enrollment wizards will have you run scripts in AWS cloudshell that try to download a teleport tarball from our usual CDN.
+That won't work with a pre-release build. You will need to modify the downloaded scripts to download the pre-release
+tarball from `cdn.cloud.gravitational.io` instead of `cdn.teleport.dev`, so change the provided script
+  `bash -c $(curl <url>)`
+  to
+  `bash -c $(curl <url> | sed 's/cdn.teleport.dev/cdn.gravitational.io/')`
+
+[^rds-enrollment-pre-release-cdn]: When you get to the "Deploy Database Service" step and click the deployment button it will create an ECS cluster and service.
```

Comment:
what image version do we expect ECS installs by default?

Reply:
it will download the version matching the proxy's version tag:

https://github.com/gravitational/teleport/blob/fa859053599524a016acbd95a9b4dd482f679e5a/lib/web/scripts/oneoff/oneoff.go#L120-L122

Reply:
on Cloud tenants with automatic updates enabled, it will be the default upgrade channel version

otherwise, it'll be the proxy version

https://github.com/gravitational/teleport/blob/fa859053599524a016acbd95a9b4dd482f679e5a/lib/web/integrations_awsoidc.go#L150-L159

On my staging tenant with a custom dev build, that deploys 16.4.2 which will also need to be changed manually for a pre-release build

Reply:
not sure if it's a good idea (as i am still learning cloud deployment), but could we maybe use some env vars to overwrite the CDN, ECS image when deploying to staging?

Reply:
yes we can do that for custom dev builds. I can make it detect a `-dev` pre-release target perhaps.
The image come from a *different* repo in that case: `<accountid>.dkr.ecr.us-west-2.amazonaws.com/teleport-local-build`

However for release testing, I'm just going to make it detect any pre-release build and switch to the staging image repo.
We can add an env var later, I want to be cautious that we don't introduce some security issue by controlling the image/cdn with an env var - let's discuss with the team before we do that.

Reply:
- https://github.com/gravitational/cloud-terraform/pull/5642
- https://github.com/gravitational/teleport/pull/47474